ph5/utilities/tests/test_metadatatoph5.py

```diff
@@ -51,7 +50,7 @@ def test_main(self):
         test main function
         """
         testargs = ['metadatatoph5', '-n', 'master.ph5', '-f',
-                    'ph5/test_data/metadata/station.xml']
+                    '../metadata/station.xml']
         with patch.object(sys, 'argv', testargs):
             metadatatoph5.main()
         self.assertTrue(os.path.isfile('master.ph5'))
```

Comment:
As the command in setUp(), it exclude test_main().

+ The ph5_object in setUp() is to provide the ph5_object for testing fuctions in metadatoph5.
+ The ph5_object in test_main() is different. Because main() includes create ph5_object and close it in itself. The ph5_object need to be created after main() added info and close it, so that we can use that ph5_object to read info for checking.

Reply:
I will create a different class for the test that does not need ph5_object.